packages/story-utils/src/index.ts

```diff
@@ -1,4 +1,2 @@
 export { default as generateSnapshots } from './generateSnapshots';
-export { default as getStories } from './getStories';
```

Comment:
Removed `getStories` because

- I don't think we get a lot of value out of it, anymore. It's used in Along and LP to create a basic test that all stories render without blowing up. But I think we get that anyway with Chromatic, axe-storybook-testing, etc. Once this is released I'll remove those tests from the aforementioned apps.
- I had a hard time getting the new Storybook 6.4 types to work with it. Doesn't seem like the pain of figuring that out is worth it.